DataFormats/CTPPSReco/src/classes_def.xml

```diff
@@ -90,6 +90,23 @@
   <class name="edm::DetSetVector<CTPPSPixelRecHit>"/>
   <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelRecHit> >"/>
 
+  <class name="CTPPSPixelLocalTrack" ClassVersion="3">
+    <version ClassVersion="3" checksum="166602192"/> 
+  </class>
+  <class name="edm::DetSet<CTPPSPixelLocalTrack>"/>
+  <class name="std::vector<CTPPSPixelLocalTrack>"/>
+  <class name="std::vector<edm::DetSet<CTPPSPixelLocalTrack> >"/>
+  <class name="edm::DetSetVector<CTPPSPixelLocalTrack>"/>
+  <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelLocalTrack> >"/>
+  <class name="edm::DetSetVector<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit>"/>
+  <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit>>"/>
+  <class name="std::vector<edm::DetSet<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit> >"/>
+  <class name="std::vector<CTPPSPixelLocalTrack::CTPPSPixelFittedRecHit>"/>
+  <class name="math::GlobalPoint"/>
```

Comment:
this is redundant to what's in https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/DataFormats/Math/src/classes_def.xml#L10

Reply:
@slava77 If I remove the GlobalPoint I get the following error at run time:  
[a] Fatal Root Error: @SUB=TStreamerInfo::Build
CTPPSPixelFittedRecHit: ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<float>,ROOT::Math::GlobalCoordinateSystemTag> has no streamer or dictionary, data member "space_point_on_det_" will not be saved.

space_point_on_det_ is in fact a GlobalPoint.

Btw DataFormats/Math/src/classes_def.xml is changed in master wrt the one you quote.